src/dev/build/tasks/os_packages/create_os_package_tasks.test.ts

```diff
@@ -158,6 +158,22 @@ it('runs task with the expected values for RPM x64', async () => {
             "linux",
           ],
         ],
+        Array [
```

Comment:
I may not modify tests which is unrelated to the PR changes. Just curious why changing these tests?

Reply:
Sorry about that. I'm just testing something and making the commit history kind of messy. I will change it later today.